libs/core/lci_base/src/lci_environment.cpp

```diff
@@ -115,8 +115,9 @@ namespace hpx { namespace util {
         }
         if (!is_mpi_initialized)
         {
+            int provided;
             retval =
-                MPI_Init_thread(nullptr, nullptr, MPI_THREAD_SINGLE, nullptr);
+                MPI_Init_thread(nullptr, nullptr, MPI_THREAD_SINGLE, &provided);
```

Comment:
If you initialize MPI, please use the mpi_base/mpi_environment helper functions to ensure consistent parameters (SINGLE_THREADED vs. MULTI_THREADED)

Reply:
okay, I will update it.